src/overlay/Overlay.cpp

```diff
@@ -195,12 +197,40 @@ void Overlay::Update()
         }
     }
 
+    // Display binding in bottom right corner for 5 seconds
+    if (!Bindings::IsFirstTimeSetup() && !m_bindHintShown)
```

Comment:
I would rather do something like in `feature/notifications` branch instead of this, and consider expanding it to sol_ImGui for mods to use.

Maybe even add some setting for the type of spdlog loglevel to notify about and automatically notify when someone logs something. Could be very useful for mods and also internal errors.

The branch needs tweaking, is just a starting point basically reimplementing this PR. It has issues with close button looking weird (probably collision with our style vs. default ImGui style the ImGuiNotify expects...) and is missing license in ThirdParty_LICENSE file.